lambda/locals.tf

```diff
@@ -22,16 +22,16 @@ locals {
   log_data_function_name              = "${var.project}-log-data-${local.environment}"
   notifications_function_name         = "${var.project}-notifications-${local.environment}"
   yara_av_function_name               = "${var.project}-yara-av-${local.environment}"
-  api_update_queue_name               = "tdr-api-update-${local.environment}"
+  api_update_queue_name               = "${var.project}-api-update-${local.environment}"
   api_update_queue                    = "arn:aws:sqs:${var.region}:${data.aws_caller_identity.current.account_id}:${local.api_update_queue_name}"
   api_update_queue_url                = "https://sqs.${var.region}.amazonaws.com/${data.aws_caller_identity.current.account_id}/${local.api_update_queue_name}"
-  antivirus_queue                     = "arn:aws:sqs:${var.region}:${data.aws_caller_identity.current.account_id}:tdr-antivirus-${local.environment}"
-  antivirus_queue_url                 = "https://sqs.${var.region}.amazonaws.com/${data.aws_caller_identity.current.account_id}/tdr-antivirus-${local.environment}"
-  checksum_queue                      = "arn:aws:sqs:${var.region}:${data.aws_caller_identity.current.account_id}:tdr-checksum-${local.environment}"
-  checksum_queue_url                  = "https://sqs.${var.region}.amazonaws.com/${data.aws_caller_identity.current.account_id}/tdr-checksum-${local.environment}"
-  download_files_queue                = "arn:aws:sqs:${var.region}:${data.aws_caller_identity.current.account_id}:tdr-download-files-${local.environment}"
-  download_files_queue_url            = "https://sqs.${var.region}.amazonaws.com/${data.aws_caller_identity.current.account_id}/tdr-download-files-${local.environment}"
-  file_format_queue                   = "arn:aws:sqs:${var.region}:${data.aws_caller_identity.current.account_id}:tdr-file-format-${local.environment}"
-  file_format_queue_url               = "https://sqs.${var.region}.amazonaws.com/${data.aws_caller_identity.current.account_id}/tdr-file-format-${local.environment}"
+  antivirus_queue                     = "arn:aws:sqs:${var.region}:${data.aws_caller_identity.current.account_id}:${project}-antivirus-${local.environment}"
```

Comment:
Should this not be `${var.project}-...` instead of `${project}`?

Reply:
Ah yeah sorry. I've changed it.